CRITICAL - 2020-09-14 00:00:39 --> Call to undefined method App\Models\UsuarioModel::inser
#0 D:\xampp\htdocs\sisbomberos\app\Controllers\Usuario.php(80): CodeIgniter\Model->__call('inser', Array)
#1 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(918): App\Controllers\Usuario->add()
#2 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
#3 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xampp\htdocs\sisbomberos\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-09-14 00:10:21 --> Undefined offset: 3
#0 D:\xampp\htdocs\sisbomberos\system\Validation\Rules.php(220): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined offse...', 'D:\\xampp\\htdocs...', 220, Array)
#1 D:\xampp\htdocs\sisbomberos\system\Validation\Validation.php(325): CodeIgniter\Validation\Rules->is_unique('rey', 'usuario.user', Array, NULL)
#2 D:\xampp\htdocs\sisbomberos\system\Validation\Validation.php(189): CodeIgniter\Validation\Validation->processRules('user', 'user', 'rey', Array, Array)
#3 D:\xampp\htdocs\sisbomberos\system\Model.php(1505): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#4 D:\xampp\htdocs\sisbomberos\system\Model.php(727): CodeIgniter\Model->validate(Array)
#5 D:\xampp\htdocs\sisbomberos\app\Controllers\Usuario.php(80): CodeIgniter\Model->insert(Array)
#6 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(918): App\Controllers\Usuario->add()
#7 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
#8 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\sisbomberos\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-09-14 00:17:39 --> Undefined offset: 3
#0 D:\xampp\htdocs\sisbomberos\system\Validation\Rules.php(220): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined offse...', 'D:\\xampp\\htdocs...', 220, Array)
#1 D:\xampp\htdocs\sisbomberos\system\Validation\Validation.php(325): CodeIgniter\Validation\Rules->is_unique('rey', 'usuario.user', Array, NULL)
#2 D:\xampp\htdocs\sisbomberos\system\Validation\Validation.php(189): CodeIgniter\Validation\Validation->processRules('user', 'user', 'rey', Array, Array)
#3 D:\xampp\htdocs\sisbomberos\system\Model.php(1505): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#4 D:\xampp\htdocs\sisbomberos\system\Model.php(727): CodeIgniter\Model->validate(Array)
#5 D:\xampp\htdocs\sisbomberos\app\Controllers\Usuario.php(80): CodeIgniter\Model->insert(Array)
#6 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(918): App\Controllers\Usuario->add()
#7 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
#8 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\sisbomberos\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-09-14 00:38:07 --> syntax error, unexpected 'null' (T_STRING), expecting ')'
#0 D:\xampp\htdocs\sisbomberos\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('D:\\xampp\\htdocs...')
#1 D:\xampp\htdocs\sisbomberos\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('CodeIgniter\\Val...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('CodeIgniter\\Val...')
#3 D:\xampp\htdocs\sisbomberos\system\Validation\Validation.php(592): spl_autoload_call('CodeIgniter\\Val...')
#4 D:\xampp\htdocs\sisbomberos\system\Validation\Validation.php(147): CodeIgniter\Validation\Validation->loadRuleSets()
#5 D:\xampp\htdocs\sisbomberos\system\Model.php(1505): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#6 D:\xampp\htdocs\sisbomberos\system\Model.php(727): CodeIgniter\Model->validate(Array)
#7 D:\xampp\htdocs\sisbomberos\app\Controllers\Usuario.php(80): CodeIgniter\Model->insert(Array)
#8 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(918): App\Controllers\Usuario->add()
#9 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
#10 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\sisbomberos\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2020-09-14 00:39:08 --> syntax error, unexpected 'null' (T_STRING), expecting ')'
#0 D:\xampp\htdocs\sisbomberos\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('D:\\xampp\\htdocs...')
#1 D:\xampp\htdocs\sisbomberos\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('CodeIgniter\\Val...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('CodeIgniter\\Val...')
#3 D:\xampp\htdocs\sisbomberos\system\Validation\Validation.php(592): spl_autoload_call('CodeIgniter\\Val...')
#4 D:\xampp\htdocs\sisbomberos\system\Validation\Validation.php(147): CodeIgniter\Validation\Validation->loadRuleSets()
#5 D:\xampp\htdocs\sisbomberos\system\Model.php(1505): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#6 D:\xampp\htdocs\sisbomberos\system\Model.php(727): CodeIgniter\Model->validate(Array)
#7 D:\xampp\htdocs\sisbomberos\app\Controllers\Usuario.php(80): CodeIgniter\Model->insert(Array)
#8 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(918): App\Controllers\Usuario->add()
#9 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
#10 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\sisbomberos\index.php(45): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2020-09-14 01:46:33 --> syntax error, unexpected '$id' (T_VARIABLE), expecting ')'
#0 D:\xampp\htdocs\sisbomberos\system\Router\Router.php(195): CodeIgniter\Router\Router->autoRoute('Usuario/viwagre...')
#1 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(793): CodeIgniter\Router\Router->handle('Usuario/viwagre...')
#2 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(364): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xampp\htdocs\sisbomberos\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-09-14 10:48:03 --> Undefined variable: totalTime
#0 D:\xampp\htdocs\sisbomberos\system\Debug\Toolbar\Views\toolbar.tpl.php(56): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'D:\\xampp\\htdocs...', 56, Array)
#1 D:\xampp\htdocs\sisbomberos\system\Debug\Toolbar.php(498): include('D:\\xampp\\htdocs...')
#2 D:\xampp\htdocs\sisbomberos\system\Debug\Toolbar.php(455): CodeIgniter\Debug\Toolbar->format(Array, 'html')
#3 D:\xampp\htdocs\sisbomberos\app\Config\Events.php(50): CodeIgniter\Debug\Toolbar->respond()
#4 [internal function]: CodeIgniter\Events\Events::Config\{closure}()
#5 D:\xampp\htdocs\sisbomberos\system\Events\Events.php(188): call_user_func(Object(Closure))
#6 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(293): CodeIgniter\Events\Events::trigger('pre_system')
#7 D:\xampp\htdocs\sisbomberos\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-09-14 10:56:50 --> Trying to get property 'user' of non-object
#0 D:\xampp\htdocs\sisbomberos\app\Models\UsuarioModel.php(59): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'D:\\xampp\\htdocs...', 59, Array)
#1 D:\xampp\htdocs\sisbomberos\app\Controllers\Usuario.php(89): App\Models\UsuarioModel->validarUserUpdate('rey67')
#2 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(918): App\Controllers\Usuario->add()
#3 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
#4 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 D:\xampp\htdocs\sisbomberos\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-09-14 11:03:32 --> 'break' not in the 'loop' or 'switch' context
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-09-14 11:08:49 --> Undefined variable: validarUserUpdate
#0 D:\xampp\htdocs\sisbomberos\app\Controllers\Usuario.php(90): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'D:\\xampp\\htdocs...', 90, Array)
#1 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(918): App\Controllers\Usuario->add()
#2 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
#3 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xampp\htdocs\sisbomberos\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-09-14 11:32:58 --> Undefined property: stdClass::$user
#0 D:\xampp\htdocs\sisbomberos\app\Models\UsuarioModel.php(59): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'D:\\xampp\\htdocs...', 59, Array)
#1 D:\xampp\htdocs\sisbomberos\app\Controllers\Usuario.php(89): App\Models\UsuarioModel->validarUserUpdate('26', 'rey44')
#2 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(918): App\Controllers\Usuario->add()
#3 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
#4 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 D:\xampp\htdocs\sisbomberos\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-09-14 12:19:17 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'is null' at line 4
#0 D:\xampp\htdocs\sisbomberos\system\Database\MySQLi\Connection.php(331): mysqli->query('\r\n             ...')
#1 D:\xampp\htdocs\sisbomberos\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('\r\n             ...')
#2 D:\xampp\htdocs\sisbomberos\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('\r\n             ...')
#3 D:\xampp\htdocs\sisbomberos\app\Models\NoticiasModel.php(41): CodeIgniter\Database\BaseConnection->query('\r\n             ...')
#4 D:\xampp\htdocs\sisbomberos\app\Controllers\Noticias.php(12): App\Models\NoticiasModel->total_n()
#5 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(918): App\Controllers\Noticias->index()
#6 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Noticias))
#7 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\sisbomberos\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-09-14 15:08:23 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'deleted_at is null and email="rdsangamat@alumno.unsm.edu.pe"' at line 1
#0 D:\xampp\htdocs\sisbomberos\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT * FROM u...')
#1 D:\xampp\htdocs\sisbomberos\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM u...')
#2 D:\xampp\htdocs\sisbomberos\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM u...')
#3 D:\xampp\htdocs\sisbomberos\app\Models\UsuarioModel.php(47): CodeIgniter\Database\BaseConnection->query('SELECT * FROM u...')
#4 D:\xampp\htdocs\sisbomberos\app\Controllers\Convocatoria.php(67): App\Models\UsuarioModel->validarGmail('rdsangamat@alum...')
#5 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(918): App\Controllers\Convocatoria->registrarNew()
#6 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Convocatoria))
#7 D:\xampp\htdocs\sisbomberos\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\sisbomberos\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
